.infra/ch_sync.ts

```diff
@@ -0,0 +1,287 @@
+import * as gcp from '@pulumi/gcp';
+import * as k8s from '@pulumi/kubernetes';
+import {readFile} from "fs/promises";
+import * as pulumi from "@pulumi/pulumi";
+import {all, Input, Output, ProviderResource} from "@pulumi/pulumi";
+import {createHash} from "crypto";
+import {input as inputs} from '@pulumi/kubernetes/types';
+import {
+  createServiceAccountAndGrantRoles,
+  PodResources,
+  stripCpuFromLimits
+} from "@dailydotdev/pulumi-common";
+
+
+type ClickhouseSyncArgs = {
+  propsPath: string;
+  propsVars: Record<string, Input<string>>;
+  version?: string;
+  limits?: PodResources;
+  env?: Input<k8s.types.input.core.v1.EnvVar>[];
+  disableHealthCheck?: boolean;
+};
+
+
+export function deployClickhouseSync(
+  name: string,
+  namespace: string | Input<string>,
+  args: ClickhouseSyncArgs,
+  {
+    limits = {
+      cpu: '1',
+      memory: '1024Mi',
+    },
+    env = [],
+    image = 'gcr.io/daily-ops/clickhouse-sink-docker:latest',
```

Comment:
it's a default. we can override it with version. since we have full control over versions, I think it's safe to use latest as default and override in specific cases (like testing)

Reply:
every time i see `:latest` i get the chills. Have bad experience with it. I'd rather have it fixed to something by default and if needed we can use latest in specific cases. otherwise we're risking cache issues

Reply:
Why do we build our own version of the clickhouse-sink, when they publish their own images?

https://hub.docker.com/layers/altinity/clickhouse-sink-connector/2.0.1-lt/images/sha256-99f38d1b5c34ac8bbcc86a86ca1eadca34e984c77ff1e2b3a54d5ee612029037

Reply:
In #1858 I used the upstream image